resource-watch/resource-watch
# Save & Remove from favorites

## Overview
Dont use popup to save @ collection when clicking favorites icon, just save to favorites or remove directly. 

Bonus
+ generic linting stuff
+ Cleanup logic
+ use `response.ok` instead of checking for **200** status code for favorite requests

## Testing instructions
1. Make sure you can save datasets & widgets to favorites
2. Make sure you can remove them also

There should never appear any collection popup! 

## Pivotal task
https://www.pivotaltracker.com/story/show/156256396
